CommandLineClient/src/main/java/com/dslplatform/compiler/client/parameters/DslCompiler.java

```diff
@@ -535,23 +535,23 @@ private static ByteStream getByteStream(Context context) {
 			final Context context,
 			final File compiler,
 			final List<String> arguments) throws ExitException {
-		final Either<Utils.CommandResult> result;
+		Either<Utils.CommandResult> result;
 		if (Utils.isWindows()) {
 			result = Utils.runCommand(context, compiler.getAbsolutePath(), compiler.getParentFile(), arguments);
 		} else {
 			final Either<String> mono = Mono.findMono(context);
 			if (mono.isSuccess()) {
 				arguments.add(0, compiler.getAbsolutePath());
-				final Either<Utils.CommandResult> firstTry = Utils.runCommand(context, mono.get(), compiler.getParentFile(), arguments);
-				if (!firstTry.isSuccess()) {
-					context.warning("Running Mono failed. Mono is buggy, so retry is in order... in 1 second");
-					try {
-						Thread.sleep(1000);
-					} catch (InterruptedException ignore) {
+				int retryCount = 0;
+				result = Utils.runCommand(context, mono.get(), compiler.getParentFile(), arguments);
```

Comment:
I don't think we need to try 4 times. One retry sounds good enough, but maybe we should prompt the user if prompt context is available